src/payment/strategies/opy/opy-payment-strategy.ts

```diff
@@ -86,4 +107,21 @@ export default class OpyPaymentStrategy implements PaymentStrategy {
     deinitialize(_options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
         return Promise.resolve(this._store.getState());
     }
+
+    private async _installWidget(containerId: string, config: OpyWidgetConfig): Promise<void> {
+        const widgetContainer = document.getElementById(containerId);
+
+        if (widgetContainer) {
+            try {
+                const widgets = await this._scriptLoader.loadOpyWidgets(config.region);
```

Comment:
Same as before is this supposed to be widgets or a widget?

Reply:
Their library is in plural, but I'll make it singular